autonomy/deploy/image.py

```diff
@@ -78,6 +78,16 @@ def build_image(agent: PublicId, pull: bool = False, dev: bool = False) -> None:
     )
 
     for stream_obj in stream:
-        stream_data = json.loads(stream_obj.decode())
-        if "stream" in stream_data:
-            print("[docker]" + stream_data["stream"], end="")
+        for line in stream_obj.decode().split("\n"):
+            line = line.strip()
```

Comment:
This will take care of both `\r` and `\n`, since we're splitting using `\n` and using `strip` to get rid of `\r`.